src/resolvers/user.js

```diff
@@ -68,27 +68,23 @@ export default {
     updateUser: combineResolvers(
       isAuthenticated,
       async (parent, { username }, { models, me }) => {
-        const user = await models.User.findById(me.id);
-        return await user.update({ username });
+        console.log({models: models, me})
+        return  await models.User.findByIdAndUpdate(me.id, { username: username}, {new: true});
```

Comment:
Do the shorthand object notation:

`models.User.findByIdAndUpdate(me.id, { username }, { new: true });` instead of `models.User.findByIdAndUpdate(me.id, { username: username }, { new: true });`

Reply:
fixed